README.md

```diff
@@ -4,233 +4,6 @@
 
 Package that contains libraries to display [soccer_vision_3d_msgs](https://index.ros.org/p/soccer_vision_3d_msgs/) as RViz Markers.
 
-The node simply listens for ball, field boundary, goalposts, markings, obstacles and robots,
-converts them to visualization markers and publishes them so they can be visualized in RViz.
+This package is only available currently for ROS2 Rolling.
```

Comment:
```suggestion
This package is currently only available for ROS2 Rolling.
```
For me, this feels more correct, but only I speak English second as language 😉

Reply:
Just to clarify, I didn't know if I was right or wrong with my suggestion, it was just a feeling.
The comment about English being my second language was just supposed to mean, that I was not sure. It was in no way meant as a comparison or affront.

I felt the need to clarify, as I noticed, I didn't really convey the reason for my last comment:

> For me, this feels more correct, **but only I speak English second as language wink**

Reply:
Both are correct, but your suggestion is more natural!